### ISSUE TITLE
Add rclone Support

### ISSUE THREAD

**Issue by adan89lion** [zhanghai/MaterialFiles#215]
It would be efficient when rclone feature ported into the app (also partially solving other issues like WebDAV support and cloud storage support, because rclone covers all of them)

Similar concept has been implemented by another developer on GitHub (rcloneExplorer)

Possible Roadmap:
1. Availability to import rclone config file (rclone.conf). Users might need to configure on other places (e.g. Termux or other CLI interface)
2. Copy / Move / Sync files and directories between local storage (Internal Storage, SD Card and USB OTGs) to remote storage (provided by rclone.conf)
3. Also it will be privacy friendly when users can upload files to an encrypted storage.

**Comment by Brajesh3** [comment]
Any update?

**Comment by niksingh710** [comment]
yeah addition of this will be nice as for now `rcx` is only the available app to be used and it seemed to be outdated.

**Comment by alexanderadam** [comment]
> `rcx` is only the available app to be used and it seemed to be outdated.

[RoundSync](https://github.com/newhinton/Round-Sync/#round-sync---rclone-for-android) is a maintained fork with bugfixes and new features (i.e. regular one way sync).
It also integrates well with MaterialFiles.

[The documentation](https://roundsync.com/) has more information.

**Comment by niksingh710** [comment]
> [The documentation](https://roundsync.com/) has more information.

SAF/Android File explorer section is blank there

**Comment by alexanderadam** [comment]
> > [The documentation](https://roundsync.com/) has more information.
> 
> SAF/Android File explorer section is blank there

It probably got lost [when migrated from RCX](https://x0b.github.io/docs/#adding-local-storage-saf).
I'm sure the maintainer would be happy if someone would chime in [to fix this](https://github.com/newhinton/round-sync-documentation/blob/master/src/usage/saf.md).

**Comment by Brajesh3** [comment]
## [chenxiaolong/RSAF](https://github.com/chenxiaolong/RSAF/tree/607ec2e75f0d9f49c09d23741e739e2af8a209f7)
An Android Storage Access Framework document provider for rclone


It would be good to have **automatic backup mechanism** in material files alltogether
